graspologic/nominate/VNviaSGM.py

```diff
@@ -0,0 +1,275 @@
+import numpy as np
+from ..match import GraphMatch as GMP
+from sklearn.base import BaseEstimator
+import itertools
+
+
+class VNviaSGM(BaseEstimator):
+    """
+    This class implements vertex nomination via seeded graph matching
+
+    Parameters
+    ----------
+    G_1: 2d-array, square
+        Adjacency matrix of `G_1`, the graph where voi is known
+
+    G_2: 2d-array, square
+        Adjacency matrix of `G_2`, the graph where voi is not known
+
+    h: int
+        distance between voi used to create induced subgraph on `G_1`
+
+    ell: int
+        distance from seeds to other verticies to create induced subgraphs on `G_1`
+
+    R: int
+        Number of restarts for soft seeded graph matching algorithm
+
+    Attributes
+    ----------
+    n_seeds_used: int
```

Comment:
This is a necessary parameter because it is not guaranteed that all the seeds will be in the induced subgraph about the voi. As a result, this parameter tells you how many seeds were used in generating the induced subgraphs that are matched with seeded graph matching.